### PR TITLE
Test pwa

### PR DESCRIPTION
**Pull request by LWcaveman**
In this version, there is a working service worker and support for adding a book to indexedDB and getting a book from IndexedDB

### CODE REVIEW

File: server/server.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 require('dotenv').config();
+const { default: axios } = require('axios');
```
Comment (Thunderpig851) [inline server/server.js:2]: Any reason for the destructuring on axios? I don't think I've ever seen this.

**Review by Thunderpig851 — APPROVED** [review]
Looks good. \My only question is on the destructuring use on the axios requirement call at the top of server.js

**Review by jck85 — APPROVED** [review]
Awesome!

File: server/server.js (hunk 1 above)
Comment (LWcaveman) [reply server/server.js:2]: Hum, I guess it got automatically added when I wrote the Axios request. I don't remember adding it and I have never seen that either.

**Pull request merged**